ptx/sec_graph_sketch.ptx

```diff
@@ -1009,7 +1009,7 @@
     In <xref ref="fig_sage_sinx"/>,
     two graph of <m>y=\sin(x)</m> is given,
     generated by <pubtitle>Sage</pubtitle> and <pubtitle>Mathematica</pubtitle>.
-    The small points represent each of the places where each <init>CAS</init> sampled the function.
+    The small points represent each of the places where each <!-- <init>CAS</init> --> CAS sampled the function.
     Notice how at the <q>bends</q>
```

Comment:
I probably fixed a problem in a wrong way. 
For some reason, that markup displays CAS as "cas" in the pdf. I supposed the way PreTeXt handles it should change, right?  This also appears in Example 3.5.8, though I missed fixing it there.
At the same time, I wasn't sure of the point of the init markup. My brief search for that markup just led me to running "pretext init" which wasn't helpful.

Reply:
Ahh.... I think the problem is that the Carlito font set doesn't include small caps. I will ask if there is a good way to work around this.

Reply:
There is a [font based on Carlito](https://codeberg.org/yomli/latte-fonts) that includes small caps, but I didn't figure out how to get LaTeX to use it.

Reply:
I think we should keep markup like `init` because it will produce semantic tags in HTML, which is useful for those needing a screen reader. (It should know to pronounce each letter rather than trying to make it a word.)

Options:
1. Override templates in LaTeX so that small caps are regular caps
2. Change to a font that supports small caps
3. Keep the current font, but use an alternate font for small caps

Let me know what you prefer.